plugins/CoreAdminHome/Controller.php

```diff
@@ -320,21 +320,34 @@ public function trackingCodeGenerator()
      */
     public function optOut()
     {
+        $reref = Common::getRequestVar('reref', false);
+        if (!empty($reref)) {
+            Url::redirectToUrl($reref);
```

Comment:
here is a security vulnerability, "Open redirect": https://www.owasp.org/index.php/Open_redirect
 could it be avoided?